### PR TITLE
OSDOCS-5693: Update RHDE page for 4.13

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4

Issue:
https://issues.redhat.com/browse/OSDOCS-5693

Link to docs preview:
http://file.rdu.redhat.com/shdiaz/OSDOCS-5693/overview/rhde-overview.html

QE review:
- [ ] QE has approved this change.

Additional information:
Also deletes files inadvertently included in original set-up of branch, but that are not part of the topic map for RHDE.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@DanielFroehlich and @dhellmann Our cover page for RHDE to go out with MicroShift 4.13. PTAL. Now is a good time to discuss any additions we want to try out with 4.13.

File: modules/about-rhde.adoc [hunk 1]
```diff
@@ -17,28 +17,34 @@ image::RHDevice_Edge_Overview_2.png[]
...
 
-To find the verified release number for each {product-title} product, see the following information:
+The two products that make up {product-title} work together based on compatible versions. To successfully pair your products, use the verified release number for each as listed in the following table:
 
-[cols="1,1"]
+[cols="3"]
 |===
-|{microshift}
-|{op-system-ostree}
+|*Release status*
```
Comment (dhellmann) [inline modules/about-rhde.adoc:24]: We should clarify that this is the release status of MicroShift. RHEL is GA.

**Comment by dhensel-rh** [comment]
@ShaunaDiaz Near the bottom of the Red Hat Device Edge overview document are two links.

[Red Hat build of MicroShift](https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.13)

Those links point to a web page that is  404 Page Not found. Is this link mean to point to a working web page ?

**Comment by ShaunaDiaz** [comment]
> @ShaunaDiaz Near the bottom of the Red Hat Device Edge overview document are two links.
> 
> [Red Hat build of MicroShift](https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.13)
> 
> Those links point to a web page that is 404 Page Not found. Is this link meant to point to a working web page ?

@dhensel-rh The RHEL release notes links work. The MicroShift ones will not work until those docs are published. So those can be considered as FPO (for placement only). I'll come back and verify the page links once everything is published before I close the Jira issue.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM
Good corrections, @ShaunaDiaz !

**Review by jldohmann — APPROVED** [review]
LGTM

**Comment by jldohmann** [comment]
whoops, no autopick 😅 brain is on autopilot this monday

**Pull request merged**